src/raftstore/coprocessor/split_check/half.rs

```diff
@@ -47,12 +48,14 @@ impl SplitChecker for Checker {
         false
     }
 
-    fn split_key(&mut self) -> Option<Vec<u8>> {
+    fn split_keys(&mut self) -> Vec<Vec<u8>> {
```

Comment:
Seems all `Checker` return one key at most.

Reply:
This pr just makes them possible to return multiple keys. Split policies will be implemented as another PR.